src/server/helpers/memcached.js

```diff
@@ -0,0 +1,30 @@
+import Memcached from 'memcached';
+
+import { conf } from '../helpers/config';
+
+let memcached = null;
+
+const getMemcachedStub = () => {
+  return {
+    get: (key, callback) => callback(),
+    set: (key, value, lifetime, callback) => callback()
+  };
+};
+
+export const getMemcached = () => {
+  const host = conf.get('MEMCACHED_HOST') ? conf.get('MEMCACHED_HOST').split(',') : null;
```

Comment:
Should we require `MEMCACHED_HOST` to be defined in production environment?
Or is assuming that config will be complete is good enough?